cli/deduplifhirLib/utils.py

```diff
@@ -126,7 +126,7 @@ def wrapper(*args,**kwargs):
         print(f"Format is {fmt}")
         print(f"Data dir is {data_dir}")
 
-        training_df = parse_test_data('cli/deduplifhirLib/test_data.csv',marked=True)
+        training_df = parse_test_data('../cli/deduplifhirLib/test_data.csv',marked=True)
```

Comment:
Since the frontend is running the python backend from the `frontend` directory, I needed to update the path . We should probably move this file to a new location so this doesn't affect the backend library?

Reply:
It might make sense to generate a path relative to the file itself with `__file__` rather than trying to update each place that calls this

Reply:
We should move the `test_data.csv` file to the base of the repository at least. Would it make sense to move it to an 'assets' folder that electron can access? @natalialuzuriaga

Reply:
> It might make sense to generate a path relative to the file itself with `__file__` rather than trying to update each place that calls this

I don't like using that keyword since it is subject to python weirdness depending on the context in which the script is being called AFAIK.

Reply:
Sure, moving this to an assets folder works!

Reply:
Could you talk more about what you've run into with `__file__`? I haven't seen any issues with that before in a context like this, and I've usually used to to get around issues with the context